src/main/java/com/easyrest/components/administrator/AdministratorMenu.java

```diff
@@ -34,4 +35,8 @@ public void clickOnWaiters() {
         driver.findElement(waitersOrdersTab).click();
     }
 
+    public String seeWhichTabIsOpen() {
```

Comment:
Please rename the method, e.x. getSelectedTabName()

Reply:
Done